pkg/streamingpromql/functions/common.go

```diff
@@ -17,6 +19,7 @@ func DropSeriesName(seriesMetadata []types.SeriesMetadata, _ *pooling.LimitingPo
 	return seriesMetadata, nil
 }
 
+// InstantVectorFunction is a function to operate on an instant vector series.
```

Comment:
This might be a bit clearer:

```suggestion
// InstantVectorFunction is a function that takes in a instant vector and produces an instant vector.
```